src/platform/nrfconnect/wifi/ConnectivityManagerImplWiFi.cpp

```diff
@@ -173,6 +173,26 @@ void ConnectivityManagerImplWiFi::OnStationDisconnected()
     }
 }
 
+#if CHIP_DEVICE_CONFIG_ENABLE_SED
+CHIP_ERROR ConnectivityManagerImplWiFi::_GetSEDIntervalsConfig(ConnectivityManager::SEDIntervalsConfig & SEDIntervalsConfig)
+{
+    // Wi-Fi uses legacy power save mode that has fixed inactivity interval
```

Comment:
The interval is actually AP-controlled. We use constants just because we don't have an API for that yet.

Reply:
Right, I mean, that the intervals for our Wi-Fi platform are fixed and cannot be changed dynamically, like for Thread.

Reply:
It will be possible with the anticipated Wi-Fi `net_mgmt `API extensions. Then we will implement the method for getting these values in the `WiFiManager`.

Reply:
Ok, I added "so far" to the comment then :)